model/src/pyrenew/latent/hospitaladmissions.py

```diff
@@ -16,54 +15,153 @@
+
+    Implements a renewal process for the expected number of hospitalizations.
+
+    Notes
+    -----
+
+    The following text was directly extracted from the wastewater model
+    documentation
+    (`link <https://github.com/cdcent/cfa-forecast-renewal-ww/blob/a17efc090b2ffbc7bc11bdd9eec5198d6bcf7322/model_definition.md#hospital-admissions-component> `_).
+
+    Following other semi-mechanistic renewal frameworks, we model the _expected_
+    hospital admissions per capita :math:`H(t)` as a convolution of the
+    _expected_ latent incident infections per capita :math:`I(t)`, and a
+    discrete infection to hospitalization distribution :math:`d(\tau)`, scaled
+    by the probability of being hospitalized :math:`p_\mathrm{hosp}(t)`.
+
+    To account for day-of-week effects in hospital reporting, we use an
+    estimated _weekday effect_ :math:`\omega(t)`. If :math:`t` and :math:`t'`
+    are the same day of the week, :math:`\omega(t) = \omega(t')`. The seven
+    values that :math:`\omega(t)` takes on are constrained to have mean 1.
+
+    .. math::
+
+        H(t) = \omega(t) p_\mathrm{hosp}(t) \sum_{\\tau = 0}^{T_d} d(\tau) I(t-\tau)
+
+    Where :math:`T_d` is the maximum delay from infection to hospitalization
+    that we consider.
+    """
 
     def __init__(
         self,
-        inf_hosp_int: ArrayLike,
+        infection_to_admission_interval: RandomVariable,
         infections_varname: str = "infections",
-        IHR_obs_varname: str = "IHR",
         hospitalizations_predicted_varname: str = "predicted_hospitalizations",
-        IHR_dist: dist.Distribution = dist.LogNormal(jnp.log(0.05), 0.05),
+        infect_hosp_rate_dist: RandomVariable = InfectHospRate("IHR"),
+        weekday_effect_dist: RandomVariable = DeterministicVariable((1,)),
+        p_hosp_dist: RandomVariable = DeterministicVariable((1,)),
     ) -> None:
         """Default constructor
 
         Parameters
         ----------
-        inf_hosp_int : ArrayLike
-            pmf for reporting (informing) hospitalizations.
+        infection_to_admission_interval : RandomVariable
+            pmf for reporting (informing) hospitalizations (see
+            pyrenew.observations.Deterministic).
         infections_varname : str
             Name of the entry in random_variables that holds the vector of
             infections.
-        IHR_obs_varname : str
-            Name of the entry in random_variables that holds the observed IHR
+        infect_hosp_rate_varname : str
+            Name of the entry in random_variables that holds the observed
+            infection-hospitalization rate (IHR).
             (if available).
         hospitalizations_predicted_varname : str
             Name to assign to the deterministic component in numpyro of
             predicted hospitalizations.
-        IHR_dist : dist.Distribution, optional
+        infect_hosp_rate_dist : RandomVariable, optional
             Infection to hospitalization rate pmf.
```

Comment:
Not a PMF but rather a distribution from which to sample the IHR

Reply:
Yes, agree. I think this should be part of the language issue #50. I'm sure there's more of this around it. I'll mark it as resolve, but add an explicit reference of this to the issue.